### ISSUE TITLE
WhithinGeoRange produces invalid graphql

### ISSUE THREAD

**Issue by dudanogueira** [weaviate/weaviate-python-client#419]
While using WithinGeoRange, the python client (version 3.22.1) will return an invalid graphql.
[Reference thread](https://forum.weaviate.io/t/geocoordinate-filtering-with-python-client-library/458/2)

```python
where_filter = {
  'operator': 'WithinGeoRange',
  'valueGeoRange': {
    'geoCoordinates': {
      'latitude': 10.8064,
      'longitude': 106.7323,
    },
    'distance': { 'max': 2000 }
  },
  'path': ['geoLocation']
}
response = (
  client.query
  .get("Place", ["postgresId", "address"])
  .with_additional('id')
  .with_where(where_filter)
  .build() # replaced here from do() to build()
)
response
```

This will output:
```
{Get{Place(where: {path: [“geoLocation”] operator: WithinGeoRange valueGeoRange: {“geoCoordinates”: {“latitude”: 10.8064, “longitude”: 106.7323}, “distance”: {“max”: 2000}}} ){postgresId address _additional {id }}}}
```
The expected output is something similar to:
```
{
    Get {
      Place(where: {
        operator: WithinGeoRange,
        valueGeoRange: {
          geoCoordinates: {
            latitude: 17.8608,    # latitude
            longitude: 41.5090    # longitude
          },
          distance: {
            max: 200000        # distance in meters
          }
        },
        path: ["geoLocation"] # property needs to be of geoLocation type.
      }) {
        postgresId
        address
        geoLocation {
          latitude
          longitude
        }
      }
    }
  }
```

**Comment by samthedataman** [comment]
are you guys going to get to this?

**Comment by tsmith023** [comment]
Hi @samthedataman, this should have been fixed in `3.23.0`. Are you still seeing the problem? Are you able to produce a minimum reproducible example that we can work from? Cheers!
